### ISSUE TITLE
aria-label

### ISSUE THREAD

**Issue by juliensl** [TeamWertarbyte/material-ui-search-bar#75]
Hi ! 

We cannot add an aria-label for accessibility.
Can you add this property please ? :) 

Thank you for your work ! 

Julien.

**Comment by leMaik** [comment]
All undocumented props are passed through to the Input component. You should be able to use aria-label.

**Comment by juliensl** [comment]
Thank you @leMaik 
Have a good day :)
